src/pages/kyeongyeon/Commponents/Feed.js

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useRef, useState } from 'react';
+import FeedList from './FeedList';
+import InputComment from './InputComment';
+import './Feed.scss';
+
+const Feed = ({ name, src, text }) => {
+  const [commentList, setCommentList] = useState([]); //리스트를 만들어야 함, 들어온 배열을 문자열로 받겠다.
+
+  const onRemove = id => {
+    setCommentList(commentList.filter(comment => comment.id !== id));
+    //인자값으로 id를 받고, setCommentList로! commentList에 있는 애들 중에 filter로 반한을 해줄건데,
+    //반환의 조건은 누른 comment의 id와 같지 않은 comment들만 반환!
+  };
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/kyeongyeonCommentData.json') //덩어리 제이슨을 받아옴
+      .then(res => res.json()) //덩어리 제이슨을 객체 현태로 변환
+      .then(data => {
```

Comment:
```suggestion
    fetch('/data/kyeongyeonCommentData.json') //덩어리 제이슨을 받아옴
      .then(res => res.json()) //덩어리 제이슨을 객체 현태로 변환
      .then(data => {
```
목데이터 사용시 맨 처음 나오는 /는 public 폴더를 지칭하기 때문에
`localhost:3000`은 생략해도 됩니다!